project/Dependencies.scala

```diff
@@ -2,11 +2,16 @@ import sbt._
 
 object Dependencies {
   val x = List(1) // scalafix:ok
-  def scalafixVersion: String = "0.14.2"
+
+  // keep this as low as possible to ensure sbt-scalafix can be upgraded even if scalafix-interfaces cannot
+  def scalafixVersion: String =
+    "0.14.2+39-c2bf18de-SNAPSHOT" // https://github.com/scalacenter/scalafix/pull/2226
 
   val all = List(
+    "ch.epfl.scala" % "scalafix-interfaces" % scalafixVersion
+      exclude ("ch.epfl.scala", "scalafix-properties"),
```

Comment:
This is the key change in this PR: `exclude` is honored in the POM, and thus will allow a sbt-scalafix-only install to load ScalafixPlugin (since classes from interfaces are present) but fail at runtime (because properties file cannot be found).